### DISCUSSION TITLE
Is there a way to download older versions of stable diffusion directml?

### DISCUSSION THREAD

**Discussion by gimpus** [lshqqytiger/stable-diffusion-webui-directml#455] [Q&A]
i have a radeon 6600xt and when i was using older version of stable diffusion directml (i dont know the version number but it was the one around august 2023) it was working great however ever since updating to the later versions there is now a constant slog of bugs and incompatability issues that just just too finicky and over encombering to deal with (from what i have gathered the 6600xt is incompatible with the new updates). 
so i was wondering if there is an archive of older versions of software or where i can find so i cant start using the software normally again before the changes. i know it was dumb of me to update without making a backup first but i cannot find in the releases older versions surely there is an archive somewhere or someone has a copy of the older versions.

**Comment by lshqqytiger** [comment]
[1.9.0](https://github.com/lshqqytiger/stable-diffusion-webui-directml/tree/v1.9.0)
[1.8.0](https://github.com/lshqqytiger/stable-diffusion-webui-directml/tree/v1.8.0-amd)
[1.7.0](https://github.com/lshqqytiger/stable-diffusion-webui-directml/tree/v1.7.0-amd)

command line:
```
# 1.9.0
git checkout tags/v1.9.0
# 1.8.0
git checkout tags/v1.8.0-amd
# 1.7.0
git checkout tags/v1.7.0-amd
```
